hooks/tk-flame_actions.py

```diff
@@ -8,24 +8,32 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
+
 """
 Hook that loads defines all the available actions, broken down by publish type.
 """
+import collections
+import os
+import re
+
 import sgtk
 from sgtk import TankError
-import flame
 
-import os
+# First flame version with the Python API is 2018.2
+if sgtk.platform.current_engine().is_version_less_than("2018.2"):
+    flame = None
+else:
+    import flame
 
 HookBaseClass = sgtk.get_hook_baseclass()
 
 ##############################################################################################################
 # Constants to be used with Flame
 
-# Defines the Schematic Reel we will use
-# Ideally this wouldn't be hardcoded, but there exists no way currently
-# to import clips without specifying a schematic reel
-SCHEMATIC_REEL = "Schematic Reel 1"
+BATCH_ACTION = "load_setup"
```

Comment:
I'm sure I just don't know enough about how flame works, but there are 2 constants with `_batch` in the name, yet `load_setup` is the `BATCH_ACTION`. Seems confusing from an outside perspective.